ktlint-ruleset-standard/src/test/resources/spec/string-template/lint.kt.spec

```diff
@@ -15,15 +15,44 @@ fun main() {
 
     println("${h["x-forwarded-proto"] ?: "http"}")
     println("${if (diff > 0) "expanded" else if (diff < 0) "shrank" else "changed"}")
+
+    @Suppress("RemoveCurlyBracesFromTemplate")
+    println("${s0}")
+    @Suppress("RemoveCurlyBracesFromTemplate", "Unused")
+    println("${s0}")
+    @Suppress("RemoveCurlyBracesFromTemplate")
+    val t = "${s0}"
 }
 
 class B(val k: String) {
     override fun toString(): String = "${super.toString()}, ${super.hashCode().toString()}, k=$k"
+
+    @Suppress("RemoveCurlyBracesFromTemplate", "Shit")
```

Comment:
what is the second suppression here testing?

Reply:
Sorry for this, seems like hard debug time
Cleaned